pyproject.toml

```diff
@@ -3,15 +3,79 @@ requires = [
     "setuptools",
     "packaging",
     "wheel",
-    "cython>=0.29.20; python_version>='3.10'",
-    "cython>=0.29.20,<3.0.0; python_version<='3.9'",
-    # See https://numpy.org/doc/stable/user/depending_on_numpy.html for
-    # the recommended way to build against numpy's C API:
+    "cython>=0.29.20",
     "numpy>=2.0.0",
     "scipy>=1.9",
 ]
 build-backend = "setuptools.build_meta"
 
+[project]
+name = "qutip"
+dynamic = ["version"]
+description = "QuTiP: The Quantum Toolbox in Python"
+readme = "README.md"
+requires-python = ">=3.10"
+license = {file = "LICENSE.txt"}
```

Comment:
Is there a way to also include the license name here directly? It's good to have something machine / human readable in addition to just a link to the text of the license.